tests/testsuite/cargo_add/git_inferred_name_multiple/stderr.log

```diff
@@ -1,2 +1,5 @@
     Updating git repository `[ROOTURL]/git-package`
-error: multiple packages found at `[ROOTURL]/git-package`: my-package1, my-package2
+error: multiple packages found at `[ROOTURL]/git-package`:
+    my-package1, my-package2, my-package3, my-package4, my-package5, my-package6
+    my-package7, my-package8, my-package9
+To disambiguate, run `cargo add --git [ROOTURL]/git-package <crate>`
```

Comment:
Unsure what the rest of the cargo errors do but I feel like a newline here would help users read this.

Reply:
It looks like we don't usually put new lines before and after the prompt. You can search for `please` in `testsuite`.